examples/hundred-tasks/src/main/java/io/littlehorse/examples/MyWorker.java

```diff
@@ -11,7 +11,11 @@ public String task1() {
 
     @LHTaskMethod("task-2")
     public String task2() {
-        return "task-2";
+        StringBuilder out = new StringBuilder();
```

Comment:
Is this intended?

Reply:
Nope, I was doing it so that I could build up state more quickly so that the warmup replicas actually took some time to warm up. Doing that enabled me to test my changes in different scenarios.

I will revert :+1: